src/main/java/com/crud/tasks/scheduler/EmailScheduler.java

```diff
@@ -0,0 +1,36 @@
+package com.crud.tasks.scheduler;
+
+import com.crud.tasks.config.AdminConfig;
+import com.crud.tasks.domain.Mail;
+import com.crud.tasks.repository.TaskRepository;
+import com.crud.tasks.service.SimpleEmailService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class EmailScheduler {
+
+    private final static String SUBJECT = "Tasks: New Trello card";
+    private final SimpleEmailService simpleEmailService;
+    private final TaskRepository taskRepository;
+    private final AdminConfig adminConfig;
+
+    @Scheduled(cron = "0 0 10 * * *")
+    public void sendInformationEmail() {
+
+        simpleEmailService.send(
+                new Mail(
+                        adminConfig.getAdminMail(),
+                        null,
+                        SUBJECT,
+                        messageTasksCount(taskRepository.count())
+                )
+        );
+    }
+
+    private String messageTasksCount(long size) {
+        return size == 1 ? "Currentlu in database you got: " + size + " task" : "Currentlu in database you got: " + size + " tasks";
```

Comment:
Można jeszcze prościej - `"Currentlu in database you got: "` jest częscią wspólną ;)